docs/integrations/react.rst

```diff
@@ -23,7 +23,7 @@ Next configure Raven.js to use your Sentry DSN:
 At this point, Raven is ready to capture any uncaught exception.
 
 Expanded Usage
---------------
+----------------
```

Comment:
They need to match exactly, it can't be longer either.

Reply:
i don't like rst

Reply:
are you sure though? the other ones were working before

Reply:
i fixed it. bleh